deploy/olm-catalog/service-telemetry-operator/manifests/service-telemetry-operator.clusterserviceversion.yaml

```diff
@@ -175,32 +179,8 @@ spec:
   apiservicedefinitions: {}
   customresourcedefinitions:
     owned:
-    - description: Represents an instance of the Service Telemetry Framework
-      displayName: STF Cluster
-      kind: ServiceTelemetry
+    - kind: ServiceTelemetry
       name: servicetelemetrys.infra.watch
-      resources:
-      - kind: Pods
-        name: ""
-        version: v1
-      - kind: ConfigMaps
-        name: ""
-        version: v1
-      - kind: ServiceTelemetrys
-        name: servicetelemetrys.infra.watch
-        version: v1beta1
-      - kind: ReplicaSets
-        name: ""
-        version: v1
-      - kind: Deployments
-        name: ""
-        version: v1
-      - kind: Services
-        name: ""
-        version: v1
-      - kind: ServiceMonitors
-        name: servicemonitors.monitoring.coreos.com
-        version: v1
```

Comment:
I was trying to figure out why `operator-sdk generate bundle [...]` stripped all these resources out (since the docs seem to imply this isn't a managed interface?). I was going through to figure out why this was the case, and all I can think of is that because I moved some resources up to the cluster scope level in the roles.yaml that it wanted to strip these out as not being self-managed?